packages/statemanager/src/simpleStateManager.ts

```diff
@@ -163,24 +158,4 @@ export class SimpleStateManager implements EVMStateManagerInterface {
   hasStateRoot(): Promise<boolean> {
     throw new Error('Method not implemented.')
   }
```

Comment:
What do you think about making these three optional since 2 out of our 4 state managers don't implement them.